src/textfield.vue

```diff
@@ -92,6 +94,16 @@ export default {
   methods: {
     fireInputEvent: function (event) {
       this.$emit('input', event.target.value)
+    },
+    focus () {
+      this.$refs.focusTarget.focus()
+    },
+    focusAndSelect () {
```

Comment:
In fact, see [here](https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/select). The spec explicitly states `select()` will not necessarily focus the element. So someone might think `select()` could select text (e.g. for copying) but not change input focus...